astropy/time/core.py

```diff
@@ -1773,6 +1773,9 @@ def replicate(self, *args, **kwargs):
         return out
 
     def to_datetime(self):
+        """
+        Convert to `datetime.timedelta` object.
```

Comment:
I think this will fail to be converted to a link, so use double backticks instead.

Reply:
`~datetime.datetime` worked in the `to_datetime` of `Time`. That's why I added.

Reply:
oh, ok, then sorry for the noise here.

Reply:
But I updated. Should I revert it back?